src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -49,6 +49,10 @@ using namespace rapidjson;
 *
 * In the case of /v2/entities/<eid>, the entityId of 'Entity* eP' is set in
 * the service routine postEntity.
+*
+* Also, if the URI param 'options' includes the value 'keyValues', then the
```

Comment:
Maybe I'm the one that has misunderstood. 
I am treating 'metadatas inside attributes' just like 'attributes inside entities'.